cloudflare.go

```diff
@@ -138,28 +138,19 @@ func (api *API) SetAuthType(authType int) {
 // ZoneIDByName retrieves a zone's ID from the name.
 func (api *API) ZoneIDByName(zoneName string) (string, error) {
 	zoneName = normalizeZoneName(zoneName)
-	res, err := api.ListZonesContext(context.Background(), WithZoneFilters(zoneName, "", ""))
+	res, err := api.ListZonesContext(context.Background(), WithZoneFilters(zoneName, api.AccountID, ""))
 	if err != nil {
 		return "", errors.Wrap(err, "ListZonesContext command failed")
 	}
 
-	if len(res.Result) > 1 && api.AccountID == "" {
-		return "", errors.New("ambiguous zone name used without an account ID")
+	switch len(res.Result) {
+	case 0:
+		return "", errors.New("Zone could not be found")
+	case 1:
+		return res.Result[0].ID, nil
+	default:
+		return "", errors.New("Ambiguous zone name; an account ID might help")
```

Comment:
Go error messages should be lowercased

```suggestion
		return "", errors.New("ambiguous zone name; an account ID might help")
```